File: angular-frontend/src/app/_model/Product/ProductModel.ts
export class ProductModel{
    ProductId!:number;
    ProductName!:string;
    Description!:string;
    Content!:string;
    Price!:number;
    PhotoReview!:string;

    /**
     *
     */
    constructor(productId:number,productName:string,description:string,content:string,price:number,photoReview:string) {
        this.ProductId=productId;
        this.ProductName=productName
        this.Description=description
        this.Content=content
        this.Price=price
        this.PhotoReview=photoReview
    }
}

File: Domain/Contracts/Products/ProductModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Contracts.Products
{
    public class ProductModel
    {
        public int ProductId { get; set; }
        public string ProductName { get; set; }
        public string Description { get; set; }
        public string Content { get; set; }
        public decimal Price { get; set; }
        public string PhotoReview { get; set; }
    }
}
